### PR TITLE
update preview URL format

### PR DESCRIPTION
**Pull request by maliroteh-sf**

### CODE REVIEW

**Review by dbreese — APPROVED** [review]

File: src/common/PreviewUtils.ts [hunk 1]
```diff
@@ -55,18 +55,6 @@ export class PreviewUtils {
...
-     * @param compName A component route or name.
-     * @returns The updated route which now starts with `c/` if compName did not start with `c/` already.
-     */
-    public static prefixRouteIfNeeded(compName: string): string {
-        if (compName.toLowerCase().startsWith('c/')) {
-            return compName;
-        }
-        return 'c/' + compName;
-    }
-
```
Comment (sfdctaka) [inline src/common/PreviewUtils.ts:69]: Wouldn't this be a breaking change if we remove it in the main branch? What happens to the downstream module that's using it? Or do we expect it to change as well if consuming this version of lwc-dev-mobile-core?

Comment (maliroteh-sf) [reply src/common/PreviewUtils.ts:69]: We're going to take this into `browser-container` branch instead of `main` for now.

Comment (sfdctaka) [reply src/common/PreviewUtils.ts:69]: Ah, didn't see that 👍

**Review by sfdctaka — APPROVED** [review]
LGTM!

**Pull request merged**